src/plugins/intel_gpu/src/kernel_selector/jitter.cpp

```diff
@@ -190,14 +190,14 @@ std::string toCodeString(size_t val) {
 std::string toCodeString(const Tensor::Dim& dim, size_t offset, bool padded, bool pad_is_dynamic, size_t pad_offset) {
     std::string pad_str = "";
     if (padded) {
-        if (dim.pad.is_dynamic) {
+        if (pad_is_dynamic) {
```

Comment:
Could you tell me what was the situaton? 
dim.pad.is_dynamic was not set as dynamic but the dim.pad was actually dynamic?

Reply:
@yeonbok hmm, the issue happens for concat's dependency when dimension is static (the dimension on which we apply concat) and in the same time it has assigned dynamic padding (because of concat optimization). In that case during jitter code generation at line 200 we will chose else branch and try to obtain `dim.pad.Total()` for dynamic padding and get error
You can try to reproduce this situation with test from my PR

Reply:
Thanks! I will check :)